src/DeviceInterface/InterfacePrototype.js

```diff
@@ -463,15 +466,20 @@ class InterfacePrototype {
                         this.firePixel('autofill_private_address')
                         break
                     default:
-                        // Also fire pixel if user autofilled identity which uses their personal address
-                        if (this.hasLocalAddresses && (data?.emailAddress === formatDuckAddress(this.#addresses.personalAddress))) {
+                        // Also fire pixel when filling an identity with the personal duck address from an email field
+                        const checks = [
+                            subtype === 'emailAddress',
```

Comment:
This is the new bit: we only want to fire this when the autofill action was triggered from an email field (not from a name, phone or whatever).